lib/md051.js

```diff
@@ -0,0 +1,36 @@
+// @ts-check
+
+"use strict";
+
+const { pathToFileURL } = require("url");
+const fs = require("fs");
+
+const { filterTokens, addError } = require("../helpers");
+
+module.exports = {
+  "names": [ "MD051", "no-dead-relative-links" ],
+  "description": "No dead relative links",
+  "tags": [ "links" ],
+  "function": function MD051(params, onError) {
+    filterTokens(params, "inline", (token) => {
+      token.children.forEach((child) => {
+        const { lineNumber, type, attrs } = child;
+        if (type === "link_open") {
```

Comment:
Currently, it only takes into account the attribute `href` of `a` (e.g: `[README](./README.md)`) but we could also check the `src` of `img`, what do you think?

What would be needed there, to check that it is `link_open` or an image?